docs/src/modules/components/ApiPage.js

```diff
@@ -31,14 +36,28 @@ function PropsTable(props) {
             propData.description !== '@ignore' && (
               <tr key={propName}>
                 <td align="left">
-                  <span className={clsx('prop-name', propData.required ? 'required' : null)}>
-                    {propName}
-                    {propData.required ? (
-                      <sup>
-                        <abbr title="required">*</abbr>
-                      </sup>
-                    ) : null}
-                  </span>
+                  <Tooltip arrow title={propData.deprecationInfo || ''} placement="bottom-start">
```

Comment:
+1 for not using a tooltip but inlining the information inside the description. The idea would be t avoid efforts for the developer, by not having to move the mouse.

Reply:
For benchmark, see how Stripe deprecates

https://stripe.com/docs/js/deprecated/handle_card_setup_element

<img width="658" alt="Capture d’écran 2021-06-25 à 14 13 32" src="https://user-images.githubusercontent.com/3165635/123423241-94fcd700-d5bf-11eb-8c23-b5d0b0b228a6.png">

---

See how Google Maps deprecates

<img width="835" alt="Capture d’écran 2021-06-25 à 14 15 46" src="https://user-images.githubusercontent.com/3165635/123423473-e2794400-d5bf-11eb-8c76-50a3a661ecbd.png">

@danilo-leal

Reply:
interesting 🤔. let's do it on design first.

Reply:
Why not, I guess there is a balance to find between making the deprecated props visible enough to the developers, but not too distracting when interested in the other props.

Reply:
Done, I feel orange is not too distracted and not too easy to forget 😂.

Reply:
Looks great. Does it mean we can remove the Tooltip here?